src/codewhisperer/service/keyStrokeHandler.ts

```diff
@@ -27,9 +27,8 @@ export class KeyStrokeHandler {
      * Speical character which automated triggers codewhisperer
      */
     public specialChar: string
-    /**
-     * Key stroke count for automated trigger
-     */
+
+    private _on: boolean = true
```

Comment:
nit: the word "on" is often used for event-based stuff in JS. Prefer "enabled" instead

Reply:
ack

Reply:
updated